orbit/pkg/table/filevault_prk/filevault_prk_darwin.go

```diff
@@ -0,0 +1,41 @@
+//go:build darwin
+// +build darwin
+
+package filevault_prk
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/osquery/osquery-go/plugin/table"
+)
+
+// Columns is the schema of the table.
+func Columns() []table.ColumnDefinition {
+	return []table.ColumnDefinition{
+		table.TextColumn("base64_encrypted"),
+	}
+}
+
+// Generate is called to return the results for the table at query time.
+//
+// Constraints for generating can be retrieved from the queryContext.
+func Generate(ctx context.Context, queryContext table.QueryContext) ([]map[string]string, error) {
+	ctx, cancel := context.WithTimeout(ctx, 5*time.Second)
+	defer cancel()
+	cmd := exec.CommandContext(
+		ctx,
+		"bash", "-c",
+		`base64 -i /var/db/FileVaultPRK.dat`,
```

Comment:
do you have any thoughts on reading the file and doing the encoding in Go? feels way safer

Reply:
@roperzh, were you thinking of something like this or is there another command you're thinking of?

```go
	cmd := exec.CommandContext(
		ctx,
		"bash",
		"-c",
		`cat < /var/db/FileVaultPRK.dat`,
	)

	out, err := cmd.Output()
	if err != nil {
		return nil, fmt.Errorf("generate failed: %w", err)
	}
	encoded := base64.StdEncoding.EncodeToString(out)

	return []map[string]string{{"base64_encrypted": strings.TrimSpace(encoded)}}, nil
```

Reply:
@gillespi314 I was thinking of:

```go
encryptedKey, err := os.ReadFile("/var/db/FileVaultPRK.dat")
if err != nil {
 // ...
}

encoded := base64.StdEncoding.EncodeToString(encryptedKey)
```